metafacture-xml/src/main/java/org/metafacture/xml/GenericXmlHandler.java

```diff
@@ -91,13 +92,32 @@ public String getRecordTagName() {
         return recordTagName;
     }
 
+    /**
+     * On entity level namespaces can be emitted, e.g.: "foo:bar". The default is to
```

Comment:
The reference to entity is perhaps in context of metafacture too ambiguous since there are entities as concepts in formeta/metafacture.  Attributes and fields can also have namespace-prefixes.

Reply:
You are right - I updated the inline documentation.